jobs/xcat-server/templates/xcat_ctl.erb

```diff
@@ -48,6 +48,19 @@ case $1 in
       source /etc/profile.d/xcat.sh
       private_ip=`ifconfig eth0 | grep "inet addr" | cut -d ":" -f 2 | cut -d " " -f 1`
       chdef -t site master=$private_ip nameservers=$private_ip forwarders=$private_ip domain=softlayer.com
+      if [[ $(head -n1 /etc/hosts) != *"softlayer.com"* ]]; then
```

Comment:
Eventually I wonder if we could expand the SoftLayer tools CLI to have a command for this?